cmd/bundle/schema.go

```diff
@@ -19,32 +16,8 @@ func newSchemaCommand() *cobra.Command {
 	}
 
 	cmd.RunE = func(cmd *cobra.Command, args []string) error {
-		// Load embedded schema descriptions.
-		docs, err := schema.LoadBundleDescriptions()
-		if err != nil {
-			return err
-		}
-
-		// Generate the JSON schema from the bundle configuration struct in Go.
-		schema, err := schema.New(reflect.TypeOf(config.Root{}), docs)
-		if err != nil {
-			return err
-		}
-
-		// Target variable value overrides can be primitives, maps or sequences.
-		// Set an empty schema for them.
-		err = schema.SetByPath("targets.*.variables.*", jsonschema.Schema{})
```

Comment:
Note: There's a regression here. With the new schema the following variable references will be invalid:
```
targets:
  foo:
     variables:
        abc: def
```

The current method of injecting custom transformations will not work here. I'm working on a followup PR on top of this one to fix this. Meanwhile, this PR is blocked on it.

Reply:
Patched: https://github.com/databricks/cli/pull/1748